site/src/pages/docs/server-graphql.mdx

```diff
@@ -0,0 +1,102 @@
+# Running a GraphQL service
+
+<Tip>
+
+Visit [armeria-examples](https://github.com/line/armeria-examples) to find a fully working example.
+
+</Tip>
+
+Let's assume we have the following [GraphQL Schema](https://graphql.org/learn/schema/):
+
+```graphql
+type Query {
+  user(id: ID): User
+}
+
+type User {
+  id: ID
+  name: String
+}
+```
+
+The schema describes that if a client requests a `User` data using the `ID` field, the GraphQL server will
+return the `User` data. The GraphQL engine uses a [DataFetcher](https://javadoc.io/doc/com.graphql-java/graphql-java/latest/graphql/schema/DataFetcher.html)
+to resolve the ID and fetch the corresponding object. For the sake of simplicity, let's implement
+`UserDataFetcher` that stores `User`s in a `Map`:
+
+```java
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+class UserDataFetcher implements DataFetcher<User> {
+
+    private final Map<String, User> data = Map.of("1", new User("1", "hero"),
+                                                  "2", new User("2", "human"),
+                                                  "3", new User("3", "droid"));
+
+    @Override
+    public User get(DataFetchingEnvironment environment) throws Exception {
+        final String id = environment.getArgument("id");
+        return data.get(id);
+    }
+}
+```
+
```

Comment:
Could you add the dependency section for `armeria-graphql`? For example:
https://github.com/line/armeria/blob/14078714659814a8d34cc9fbca15ee011524c4dc/site/src/pages/docs/advanced-zipkin.mdx#L9-L16

Reply:
Added again 🤣